v1/sargo.json

```diff
@@ -0,0 +1,143 @@
+{
+  "name": "Google Pixel 3a",
+  "codename": "sargo",
+  "unlock": ["unlock", "downgrade_android", "support"],
+  "user_actions": {
+    "unlock": {
+      "title": "OEM unlock",
+      "description": "If you haven't done so already, make sure to OEM unlock your device first.",
+      "link": "https://source.android.com/devices/bootloader/locking_unlocking#unlocking-bootloader"
```

Comment:
What's up here? Is the installer's own unlocking failing?

Reply:
Will test unlocking asap, didn't bother tbh.

Reply:
Yeah, turns out that trying it with the device locked makes fastboot fail with:

```
Error: fastboot:flash: Error: flashing failed: Error: {"error":{"killed":false,"code":1,"signal":null,"cmd":"\"/snap/ubports-installer/307/app/resources/app.asar.unpacked/node_modules/android-tools-bin/dist/linux/x86/fastboot\" flash boot \"/home/alfred/snap/ubports-installer/307/.cache/ubports/sargo/firmware/boot.img\""},"stdout":"","stderr":"Sending 'boot_b' (65536 KB)                        OKAY [  2.030s]\nWriting 'boot_b'                                   FAILED (remote: 'Fastboot command (flash:) is not allowed when locked')\nfastboot: error: Command failed"}

```

Reply:
@NeoTheThird there is now also fastboot flashbing unlock, not unly fastboot oem unlock, did you consider this?

Reply:
@Flohack74 yes, i already had a chat with fredl. Here's the issue to follow https://github.com/ubports/ubports-installer/issues/1479